hs_core/management/commands/fix_irods_user_paths.py

```diff
@@ -1,67 +0,0 @@
-# -*- coding: utf-8 -*-
-
-"""
-Check synchronization between iRODS and Django
```

Comment:
I strongly object to this file being deleted. It is the only reason my dev environments at both RENCI and CUAHSI work. Without this, only the production hosts will work properly.